tests/integration.rs

```diff
@@ -1,8 +1,29 @@
-use std::io::{Read, Write};
+use std::env;
+use std::io::Write;
+use std::path::PathBuf;
 use std::process::{Command, Stdio};
+use tempfile::TempDir;
+
+fn canonicalize_path(path: &str) -> String {
+    PathBuf::from(path)
+        .canonicalize()
+        .unwrap()
+        .to_str()
+        .unwrap()
+        .to_string()
+}
```

Comment:
Why what does this do?

Reply:
It is used to convert a path string into its canonical (absolute and normalized) form.

It's used in `test_cd_command` because macOS returns "/private/tmp" while Linux returns "/tmp" for the same directory. 

It works by converting the input string to a PathBuf, calling canonicalize() to resolve symlinks and remove redundant components, and then converting the result back to a string.